lib/check-for-updates.js

```diff
@@ -26,7 +26,7 @@ module.exports = checkForUpdates;
 
 function readPackageJsonVersion () {
   return new Promise((resolve, reject) => {
-    Fs.readFile('package.json', (err, data) => {
+    Fs.readFile(__dirname + '/../package.json', (err, data) => {
```

Comment:
We should probably be using `path.resolve` here to help keep this windows/mac/linux friendly.